games-engines/openmw/openmw-0.48.0.ebuild

```diff
@@ -137,6 +135,11 @@ src_compile() {
 }
 
 src_test() {
+	# Lua 5.x is supported in theory, but don't work as well, the test fails
+	# Upstream recommends luajit, but it has less arch coverage
+	if [[ ${ELUA} != luajit ]]; then
+		return
+	fi
```

Comment:
I think some `elog` or `ewarn` should be used here to inform tests have been skipped.

Reply:
Done